app/src/main/java/org/simple/clinic/teleconsultlog/prescription/patientinfo/TeleconsultPatientInfoModel.kt

```diff
@@ -1,22 +1,29 @@
 package org.simple.clinic.teleconsultlog.prescription.patientinfo
 
 import org.simple.clinic.patient.PatientProfile
+import java.time.LocalDate
 import java.util.UUID
 
 data class TeleconsultPatientInfoModel(
     val patientUuid: UUID,
-    val patientProfile: PatientProfile?
+    val patientProfile: PatientProfile?,
+    val prescriptionDate: LocalDate?
```

Comment:
I looked into the later commits and saw that this date is only being used to render the current date on the UI. If this is the case, I think we can make the current date part of the factory method of the model and provide it when creating it since it is very unlikely the date will change while the medical officer is on this screen.

This will let us skip needing the effect to load the current date.

Reply:
Yes, I was considering both these usages and thought having effect is better for testing. But yeah, it doesn't make much sense to have effect that loads just the date, will switch to the factory method.